hack/generated/pkg/testcommon/kube_test_context.go

```diff
@@ -7,11 +7,14 @@ package testcommon
 
 import (
 	"context"
+	"log"
+	"testing"
 	"time"
 
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
This is already included above? use `metav1` rather than `v1`?

Reply:
Bah, gopls!